celery/app/routes.py

```diff
@@ -29,7 +29,7 @@ class MapRoute:
     def __init__(self, map):
         map = map.items() if isinstance(map, Mapping) else map
         self.map = {}
-        self.patterns = OrderedDict()
+        self.patterns = {}
```

Comment:
Instead of defining and then redefining this variable, there is no need to assign it to `self` here. I realize there is no practical impact since there is no typing check. But, _if_ `mypy` were implemented someday, it would complain.